steps/browserstack-bitrise-xcui-step/1.0.0/step.yml

```diff
@@ -0,0 +1,231 @@
+title: BrowserStack App Automate - XCUITest
+summary: |
+  Run your XCUITest tests on BrowserStack App Automate
+description: |
+  Run your XCUITest tests on BrowserStack App Automate. This step collects the built IPA from $BITRISE_IPA_PATH  and test suite from `$BITRISE_BUNDLE_PATH` Environment Variables
+website: https://github.com/browserstack/browserstack-bitrise-xcui-step
+source_code_url: https://github.com/browserstack/browserstack-bitrise-xcui-step
+support_url: https://github.com/browserstack/browserstack-bitrise-xcui-step/issues
+published_at: 2022-08-11T10:42:54.720007+05:30
+source:
+  git: https://github.com/browserstack/browserstack-bitrise-xcui-step.git
+  commit: e317013319d6fd95193b090112b0948872716961
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/browserstack/browserstack-bitrise-xcui-step
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- browserstack_username: null
+  opts:
+    category: Authentication
+    description: Username of the BrowserStack account
+    is_required: true
+    is_sensitive: true
+    summary: Username for the BrowserStack account
+    title: BrowserStack username
+- browserstack_accesskey: null
+  opts:
+    category: Authentication
+    description: Access Key of the BrowserStack account
+    is_required: true
+    is_sensitive: true
+    summary: Accesskey for the BrowserStack account
+    title: BrowserStack accesskey
+- app_ipa_path: null
+  opts:
+    category: App & Test Suite
+    description: Path of the app (.ipa) file
+    is_expand: true
+    is_required: true
+    summary: Path to the app (.ipa) file
+    title: iOS app under test
+- opts:
+    category: App & Test Suite
+    description: Path of the output bundle (.app) file. Add either the $BITRISE_TEST_BUNDLE_PATH
+      (generated from the "Xcode Build for testing for iOS" step) or the absolute
+      path of the output bundle file
+    is_expand: true
+    is_required: true
+    summary: Path of the output bundle file
+    title: XCUI test suite
+  xcui_test_suite: null
+- devices_list: null
+  opts:
+    category: Devices
+    description: "Name of one or more device-OS combination in new line \n For example:\n
+      \n `iPhone 11-13`\n \n `iPhone XS-15`\n \n [View supported device-OS combinations](https://www.browserstack.com/list-of-browsers-and-platforms/app_automate)\n"
+    is_expand: true
+    is_required: true
+    summary: |
+      Name of one or more device-OS combinations
+    title: Devices
+- instrumentation_logs: true
```

Comment:
Boolean input values should be wrapped in quotes (for historical reasons). Please use "true" in default values and in the value_options too.